toonz/sources/toonz/xshnoteviewer.cpp

```diff
@@ -475,33 +474,42 @@ NoteArea::NoteArea(XsheetViewer *parent, Qt::WFlags flags)
 
   //-----
 
-  m_flipOrientationButton->setObjectName("PushButton_NoPadding");
+  m_flipOrientationButton->setObjectName("flipOrientationButton");
   m_flipOrientationButton->setFocusPolicy(Qt::FocusPolicy::NoFocus);
   m_flipOrientationButton->setFixedSize(QSize(70, 23));
+  m_flipOrientationButton->setIconSize(QSize(40, 20));
+  QIcon flipOrientationIcon = createQIcon("fliporientation");
```

Comment:
IMO this line should be just `QIcon flipOrientationIcon;` since you don't register the resource `fliporientation.svg`.
This line seems to cause alert message as follows on the debug console :  
`There qt.svg: Cannot open file ':Resources/fliporientation.svg', because: No such file or directory`